content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate, found in the bundle
+* The signature, also found in the bundle
+* The identity used in signing
 
-* [Conaisseur](https://github.com/sse-secure-systems/connaisseur#what-is-connaisseur)
-* [Kyverno](https://kyverno.io/docs/writing-policies/verify-images/)
-* [OPA Gatekeeper](https://github.com/sigstore/cosign-gatekeeper-provider)
+The following example verifies the signature on file.txt from user "name@example.com" issued by "accounts@example.com":
 
-To learn how to use Cosign with Kubernetes, review [Kubernetes](/cosign/kubernetes/).
+```
+$ cosign verify-blob file.txt --bundle txtbundle.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
 
-## More Info
+To verify, Cosign queries the transparency log to compare the public key to what’s in Rekor, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by OpenID Connect were valid.
 
-Cosign can do much more than what is discussed here. Review more information on the commands by checking out the other sections of this site.
+### Working with containers
 
-### Other Formats
+Signing and verifying a container is similar to working with blobs.   The Cosign command to sign a container image is:
 
-Cosign is useful not only for containers and container-related artifacts; it can also be used for other file types. 
+```
+$ cosign sign <image URI>
+```
 
-To learn how to sign SBOMs, WASM modules, Tekton bundles and more, review [Signing Other Types](/cosign/other_types/). For basic blobs, review [Working with Blobs](/cosign/working_with_blobs/).
+This works the same as signing a blob, but the signature becomes attached to the container itself.
 
-### SCM Integration
+To verify a signed container image, use the following command:
 
-Cosign integrates natively with source code management (SCM) systems like GitHub and GitLab.
-You can use the official [GitHub Actions Cosign installer](https://github.com/marketplace/actions/cosign-installer) or use `cosign` to generate and work safely with [SCM secrets](/cosign/git_support/) with native API integration.
+```
+$ cosign verify <image URI>
+```
+### Signing with a generated key
 
-### Attestations
+While you do not have to use an existing key you can generate a key and sign with it or another key you may wish to use.  However, it is recommended that you use keyless signing.
 
-In addition to signatures, Cosign can be used with [In-Toto Attestations](https://github.com/in-toto/attestation).
+To generate keys using a KMS provider, you can use the cosign generate-key-pair command with the --kms flag.
```

Comment:
Let's remove the bits about KMS, and you can just use the command `cosign generate-key-pair` to generate a key

Reply:
KMS part removed.